lib/index.js

```diff
@@ -74,17 +74,10 @@ export default {
 
         // Call ERB to "de-templatize" the code
         return helpers.exec(this.erbPath, erbArgs,
-          { stdin: text, cwd: fileDir }
+          { stdin: text.replace(/<%=/g, '<%'), cwd: fileDir }
```

Comment:
Sorry, looks like the check was a _negative_ check there, `text.html.erb` is a regular ERB file, `text.html.ruby` is apparently the scope of a Rails file.

Before the modification was only being done if the scope _wasn't_ `text.html.erb` (and thus must have been `text.html.ruby`), now this modification is being done on every file.

This may be a harmless modification, I just wanted to make sure that if that text is something legitimately wrong with _non_-Rails files that it is still caught.